synapse/storage/databases/main/relations.py

```diff
@@ -446,7 +446,6 @@ def _get_thread_summary_txn(
             """
 
             txn.execute(sql, (event_id, room_id, RelationTypes.THREAD, user_id))
-            row = txn.fetchone()
```

Comment:
I'm not sure, but this was added in this PR and removed, I actually meant to rebase it out instead of being a separate commit. So I'm unsure if the ramifications really matter?

Reply:
Agreed that they don't matter---just trying to check my understanding of SQL!

Reply:
With some brief experimentation, I think it the second call would always return `None`?